easybuild/easyblocks/p/psi.py

```diff
@@ -175,12 +177,16 @@ def sanity_check_step(self):
         """Custom sanity check for PSI."""
         custom_paths = {
             'files': ['bin/psi%s' % self.version.split('.')[0]],
-            'dirs': ['include', 'share/psi'],
+            'dirs': ['include', ('share/psi', 'share/psi4')],
         }
         super(EB_PSI, self).sanity_check_step(custom_paths=custom_paths)
 
     def make_module_extra(self):
         """Custom variables for PSI module."""
         txt = super(EB_PSI, self).make_module_extra()
-        txt += self.module_generator.set_environment('PSI4DATADIR', os.path.join(self.installdir, 'share', 'psi'))
+        psi4datadir = glob.glob(os.path.join(self.installdir, 'share', 'psi*'))
+        if len(psi4datadir) != 1:
+            raise EasyBuildError("Could not determine the PSI4 data dir, there are multiple possibilities: %s"
+                                 % psi4datadir)
+        txt += self.module_generator.set_environment('PSI4DATADIR', psi4datadir[0])
```

Comment:
yes, so? My is: if not 1, stop right here. I don't see how it can get any more readable?